### PR TITLE
Miscellaneous updates in Fabric docs

### PR DESCRIPTION
**Pull request by awaelchli**
## What does this PR do?

A few smaller updates in the Fabric docs batched together in one PR. 

cc @borda @carmocca @justusschock @awaelchli

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -16,9 +16,7 @@ clean:
 	rm -rf .mypy_cache
 	rm -rf .pytest_cache
 	rm -rf ./docs/build
-	rm -rf ./docs/source-fabric/generated
-	rm -rf ./docs/source-fabric/*/generated
-	rm -rf ./docs/source-fabric/api
```
Comment (awaelchli) [inline Makefile:21]: Fixed this, as this would remove actual source files :)

File: docs/source-fabric/_templates/theme_variables.jinja [hunk 2]
```diff
@@ -2,17 +2,14 @@
...
-  'tutorials': 'https://pt-lightning.readthedocs.io/en/latest/#tutorials',
-  'previous_pytorch_versions': 'https://pt-lightning.rtfd.io/en/latest/',
-  'home': 'https://lightning.ai/',
-  'get_started': 'https://pt-lightning.readthedocs.io/en/latest/introduction_guide.html',
-  'features': 'https://pt-lightning.rtfd.io/en/latest/',
-  'blog': 'https://www.pytorchlightning.ai/blog',
-  'resources': 'https://pt-lightning.readthedocs.io/en/latest/#community-examples',
-  'support': 'https://pt-lightning.rtfd.io/en/latest/',
+  'governance': 'https://pytorch-lightning.readthedocs.io/en/latest/governance.html',
+  'docs': '',
```
Comment (awaelchli) [inline docs/source-fabric/_templates/theme_variables.jinja:6]: these will be filled once we know the actual url the docs get hosted under :)

File: docs/source-fabric/guide/trainer_template.rst [hunk 3]
```diff
@@ -1,5 +1,7 @@
+:orphan:
+
 ################
 Template Trainer
 ################
 
-Coming soon.
```
Comment (awaelchli) [inline docs/source-fabric/guide/trainer_template.rst:5]: I will do this next :))

File: docs/source-fabric/index.rst [hunk 4]
```diff
@@ -60,25 +60,39 @@ Fabric is the fast and lightweight way to scale PyTorch models without boilerpla
...
+Installation
+************
+
+Fabric ships directly with Lightning. Install it with
+
+.. code-block:: bash
+
+    pip install lightning
+
+For alternative ways to install, read the `installation guide <https://pytorch-lightning.readthedocs.io/en/stable/starter/installation.html>`_.
```
Comment (awaelchli) [inline docs/source-fabric/index.rst:94]: We can also have a separate installation page in the fabric docs, but for now, this is a quick fix just to get users started. 🛹

File: src/lightning/fabric/fabric.py [hunk 5]
```diff
@@ -608,6 +608,22 @@ def load(
...
+        """Launch and initialize all the processes needed for distributed execution.
+
+        Args:
+            function: Optional function to launch when using a spawn/fork-baded strategy, for example, when using the
+                XLA strategy (``accelerator="tpu"``). The function must accept at least one argument, to which
+                the Fabric object itself will be passed.
+            *args: Optional positional arguments to be passed to the function.
+            **args: Optional keyword arguments to be passed to the function.
+
+        Returns:
```
Comment (carmocca) [inline src/lightning/fabric/fabric.py:620]: Can you add a note mentioning that calling launch more than once is a no-op?

**Review by carmocca — APPROVED** [review]

File: src/lightning/fabric/fabric.py (hunk 5 above)
Comment (awaelchli) [reply src/lightning/fabric/fabric.py:620]: done

**Review by justusschock — APPROVED** [review]

**Review by Borda — APPROVED** [review]

**Pull request merged**